### ISSUE TITLE
Windows Portable Version Missing Keynote File Import Option

### ISSUE THREAD

**Issue by OPMUSER** [giuspen/cherrytree#1275]
**Version, Operative system**
Windows 7 64bit/CherryTree 0.99.16

**Describe the bug**
Windows Portable Version Missing Keynote File Import Option, or is the file type imported via one of the other formats.

**To Reproduce**
Steps to reproduce the behavior, just click on Import, see the screenshot below.

**Screenshots**

![image](https://user-images.githubusercontent.com/17958927/97078340-448f3d00-161e-11eb-8b0f-2d27578a07b3.png)

**Comment by txe** [comment]
There is no that option because the code was not ported from python version (maybe it is not worth).
Do you want to import from Keynote or you just inform about this?

**Comment by LordMyschkin** [comment]
I'd love to import my keynote NT note tree; is there a way to to so?

**Comment by giuspen** [comment]
Yes. While the latest 0.99.41 (C++) version does not have the import from keynote yet, the old 0.39.4 (Python) has it, so you can download the older version from https://www.giuspen.com/cherrytree/#downl and use it only to import/convert to cherrytree. Then you can get back to use the latest version.
